code/modules/antagonists/traitor/objectives/bug_room.dm

```diff
@@ -0,0 +1,187 @@
+/datum/traitor_objective_category/bug_room
+	name = "Bug Room"
+	objectives = list(
+		/datum/traitor_objective/bug_room = 1,
+		/datum/traitor_objective/bug_room/risky = 1,
+		/datum/traitor_objective/bug_room/super_risky = 1,
+	)
+
+/datum/traitor_objective/bug_room
+	name = "Bug the \[DEPARTMENT HEAD]'s office"
```

Comment:
Bikeshed: Can we just make these `%DEPARTMENT_HEAD%` like every other fake format string we have? The backslash makes this look dumb, and the `[]` can be confusing.